### ISSUE TITLE
Let's document: PipeWire

### ISSUE THREAD

**Issue by vitorhcl** [tldr-pages/tldr#11747]
### Command description

PipeWire is quite stable now and even reached 1.0 recently. For now we have just a few commands documented, it would be nice if we documented everything :smiley:

### Documentation

https://docs.pipewire.org/page_programs.html

### Platform

Linux

### Commands

- [ ] pipewire
- [X] pw-cat
- [X] pw-cli
- [ ] pw-config
- [ ] pw-dot
- [ ] pw-dump
- [ ] pw-jack
- [X] pw-link
- [X] pw-loopback
- [ ] pw-metadata
- [ ] pw-mididump
- [ ] pw-mon → #11744
- [ ] pw-profiler
- [ ] pw-reserve
- [ ] pw-top
- [ ] pw-v4l2
- [ ] spa-acp-tool
- [ ] spa-inspect
- [ ] spa-json-dump
- [ ] spa-monitor
- [ ] spa-resample

**Comment by Managor** [comment]
Add `pw-play` and `pw-record` to the list. We already have pages for them.
https://github.com/tldr-pages/tldr/blob/main/pages/linux/pw-play.md
https://github.com/tldr-pages/tldr/blob/main/pages/linux/pw-record.md

**Comment by sebastiaanspeck** [comment]
> Add `pw-play` and `pw-record` to the list. We already have pages for them.
> 
> https://github.com/tldr-pages/tldr/blob/main/pages/linux/pw-play.md
> 
> https://github.com/tldr-pages/tldr/blob/main/pages/linux/pw-record.md

`pw-play` and `pw-record` are to be included in this link https://docs.pipewire.org/page_man_pw-cat_1.html. The pages above do not have a very descriptive link IMO.

**Comment by Managor** [comment]
Yeah, the links were meant as placeholders because proper documentation didn't exist back then.

**Comment by Managor** [comment]
pw-play and pw-record should be converted to aliases.

**Comment by Managor** [comment]
I added a couple of new commands to the list that don't yet have a page

**Comment by ivanbaluta** [comment]
I noticed pipewire-pulse has a man page but isn't on the list here. Was that intentional?

**Comment by Managor** [comment]
Not intentional. It's part of a different package so listing binaries of the main packages didn't catch it.

**Comment by ivanbaluta** [comment]
@Managor I want to add pw-midi2play and pw-midi2record. These are the last aliases for pw-cat, but there is no more space for examples in pw-cat itself. How should I proceed?

**Comment by Managor** [comment]
Give them the same treatment egrep got.

**Comment by Managor** [comment]
Oh I misunderstood it's midi2, not adding more examples to the previously created pages. Do they differ any way in terms of syntax from pw-midiplay and pw-midirecord? If not, I guess it's fine to just make them into aliases without resulting examples in pw-cat.

**Comment by ivanbaluta** [comment]
Yes, they actually use different flags. `pw-midiplay` and `pw-midirecord` use `--midi`, while the `midi2` versions use `--midi-clip`.

**Comment by Managor** [comment]
Bummer. Add them as standalone pages.
